viYarn/Chat/model/EpisodeContent.swift

```diff
@@ -39,7 +45,14 @@ extension EpisodeContent {
                 }
             case "span":
                 guard let message = try? element.text() else { return nil }
-                arrMessages[arrMessages.count - 1].messages.append(message)
+                let chatElement = ChatElement(message: message, tag: element.tagName(), cellID: ChatTableViewCellMessage.cellID)
+                arrMessages[arrMessages.count - 1].messages.append(chatElement)
```

Comment:
p0:
arrMessages[arrMessages.count - 1]
to
arrMessages.lastItem or arrMessages.last